### ISSUE TITLE
Mapping of label

### ISSUE THREAD

**Issue by wsy19961129** [shuangshuangguo/caffe2pytorch-tsn#19]
hello, I am just wondering how do you map the label to one-hot encoding. Is it just sort the label(string) in alphabetical order? E.g. in UCF101, ApplyEyeMakeup=0 and ApplyLipstick=1. 
I test the model using your pretrianed weights while on UCF101/split1 is 76% and on HMDB51 is about 43.3% and I do not know what the problem is. Thanks!

**Comment by linshuheng6** [comment]
@wsy19961129 
I got the same problem that the accuracy on UCF101/split1 is only 75.53%.
Have you solved this problem?

**Comment by imnotk** [comment]
recently I try to reproduce TSN in pytorch with my own code, I found some important thing that may influence you performance and pytorch is very different with tensorflow in some extent. 
1. The data sample should be like (b,t,c,h,w) i.e (b,c,t,h,w) work worse in pytorch, you can visualize it
2. You should use origin caffe input mean and input std. I use the tensorflow hyperparameter but it doesn't work.
These reasons is why my pytorch code only get accuracy on 76%. After solving 1, I get 82% acc. After solving 2, I get 85% acc.
